dsl/seed/jenkinsfiles/Jenkinsfile.prod.prepare

```
@@ -0,0 +1,407 @@
+                            ]
+
+                            seedConfig.git.branches.add(branchConfig)
+                        }
+
+                        echo "Write seed config => ${seedConfig}"
+                        sh "rm ${seedConfigFile}"
+                        writeYaml file: seedConfigFile, data: seedConfig, overwrite: true
+
+                        pushIfChanged(env.SEED_CONFIG_FILE_GIT_BRANCH, env.SEED_CONFIG_FILE_GIT_AUTHOR_CREDS_ID, "Add productized branch ${getEcosystemProductizedBranch()} after upstream release", "Main config has been updated with release branch ${getEcosystemProductizedBranch()}")
+                    }
+                }
+            }
+        }
+
+        // 1. update seed config repo on productized branch (e.g., 1.37.x-prod)
+        //       - enable productized profile (-Dproductized) on jenkins environments branch.yaml [this MUST be executed only on config file repo, e.g., kogito-pipelines]
+        //       - disable some environments when branching as they make no sense by default [this MUST be executed only on config file repo, e.g., kogito-pipelines]
+        //       - set productized_repositories[].branch to the productized one, e.g., 1.37.x-prod
+        //       - set productized_repositories[].is_branch_config_repo to true for the config file repo
+        //       - do these changes directly on the prod branch
+
+
+        stage("Update branch config on productized branch"){
+            steps{
+                script {
+                    def prodBranch = getBranchConfigProductizedBranch()
+                    def prodRepositories = getProdRepositoriesToBranch()
+
+                    dir(checkoutMainBranchSeedConfigFileRepo()) {
+                        getOrCreateGitBranch(prodBranch, getMainBranchConfigFileGitAuthorCredentialsId())
+
+                        def branchConfigFilePath = getProdBranchConfigFilePath()
+                        def branchConfig = readYaml file: branchConfigFilePath
+
+                        branchConfig.productized_repositories = prodRepositories
+                        branchConfig.productized_branch = true
+
+                        // disable some environments when branching as they make no sense by default
+                        branchConfig.environments.each { envName, envData ->
+                            if (envName.startsWith('quarkus')) {
+                                if (envName != 'quarkus-branch') {
+                                    envData.enabled = false // Keep only quarkus-branch
+                                }
+                            }
+                        }
+
+
+                        // enable productized profile
+                        def buildMvnOpts = "-Dproductized ${getAdditionalDefaultBuildMavenOpts()} "
+
+                        branchConfig.environments.each { envName, envData ->
+                            envData.env_vars.BUILD_MVN_OPTS = buildMvnOpts + (envData.env_vars.BUILD_MVN_OPTS ?: '')
```

```suggestion
                            envData.env_vars = envData.env_vars ?: [:]
                            envData.env_vars.BUILD_MVN_OPTS = buildMvnOpts + (envData.env_vars.BUILD_MVN_OPTS ?: '')
```
make sure `env_vars` is existing